packages/e2e/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@cardano-sdk/e2e",
+  "version": "0.0.1",
+  "description": "e2e tests in other packages",
+  "engines": {
+    "node": "^14"
+  },
+  "main": "dist/index.js",
+  "repository": "https://github.com/input-output-hk/cardano-js-sdk/packages/e2e",
+  "author": "Michael Chappell <martynas.kazlauskas@iohk.io>",
+  "license": "MPL-2.0",
+  "scripts": {
+    "test": "ENVFILE=.env yarn test:chrome",
```

Comment:
In order to run the workspace package.json scripts we need to include:

```
    "build": "shx echo 'build' command not implemented",
    "coverage": "shx echo No coverage report for this package",
    "cleanup": "shx rm -rf node_modules",
    "test:e2e": "yarn test && yarn test:chrome",
    "tscNoEmit": "shx echo typescript --noEmit command not implemented yet",
 ```
 ... plus the devDependency: `"shx": "^0.3.3"`